.github/workflows/stale.yml

```diff
@@ -0,0 +1,76 @@
+name: Comment on stale issues
+
+on:
+  schedule:
+  - cron: '0 0 * * *' # This will run daily at midnight
+  workflow_dispatch:
+
+jobs:
+  comment-stale-issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      contents: read
+    env:
+      LABEL_OWNER_MAPPING_JSON: >
+        {
+          "Debugger": "@wardengnaw",
+          "Debugger-ExpressionEvaluation": "@wardengnaw",
+          "Debugger-FixedPendingInsertion": "@wardengnaw",
+          "Debugger-Install": "@wardengnaw",
+          "Debugger-Poachable": "@wardengnaw",
+          "Documentation": "@mikadumont",
+          "Project System": "@jasonmalinowski",
+          "Question": "@mikadumont",
+          "Razor": "@phil-allen-msft",
+          "Unity": "@jbevain"
```

Comment:
At some point it might be useful to create (or re-use) a few different github groups here.  For example we could have a debugger set, razor set, unity set, etc.  Then we can live update the groups without needing to modify this.
Totally fine with this as v1 though.